packages/ag-charts-types/src/presets/sparkline/sparklineAxisOptions.ts

```diff
@@ -28,17 +28,17 @@ export interface AgSparklineContinuousAxisOptions<TDatum extends Date | number>
 }
 
 export interface AgSparklineCategoryAxisOptions extends AgSparklineBaseCategoryAxisOptions {
-    type: 'category';
+    type?: 'category';
```

Comment:
I'd prefer we kept these required, what was the motivation here?

Reply:
Reverted